lib/php/webhook-bundle/composer.json

```
@@ -9,10 +9,10 @@
     "license": "MIT",
     "require": {
         "php": "^8.3",
-        "symfony/messenger": "^6",
-        "guzzlehttp/guzzle": "^7.2",
         "ramsey/uuid-doctrine": "^1.5",
-        "symfony/framework-bundle": "^6"
+        "symfony/framework-bundle": "^6",
+        "symfony/http-client": "^7.2",
```

We still are on v6. It means you have tested nor run `composer update` in databox?

```suggestion
        "symfony/http-client": "^6.4",
```